bootstrap/kubeadm/config/manager/manager.yaml

```diff
@@ -22,7 +22,7 @@ spec:
             - "--leader-elect"
             - "--diagnostics-address=${CAPI_DIAGNOSTICS_ADDRESS:=:8443}"
             - "--insecure-diagnostics=${CAPI_INSECURE_DIAGNOSTICS:=false}"
-            - "--feature-gates=MachinePool=${EXP_MACHINE_POOL:=false},KubeadmBootstrapFormatIgnition=${EXP_KUBEADM_BOOTSTRAP_FORMAT_IGNITION:=false}"
+            - "--feature-gates=MachinePool=${EXP_MACHINE_POOL:=true},KubeadmBootstrapFormatIgnition=${EXP_KUBEADM_BOOTSTRAP_FORMAT_IGNITION:=false}"
```

Comment:
Would it be better to remove the `MachinePool` flag altogether in these cases?

Pro: It makes it more obvious that the feature is enabled, and suggests to a user how to disable it.

Con: It's redundant since the feature defaults to `true`, and removing clutter is good.

Reply:
I think the more idiomatic thing to do that I've seen in k/k is to remove clutter as part of the matriculation process of a feature from alpha-->GA. So it seems like we should remove it from this spec. (As long as we have UT to prove that when you invoke w/out the explicit feature flag config it is enabled.)

Reply:
Never mind, we need to leave bits in so `EXP_MACHINE_POOL` is still respected, at least for now.

Reply:
Makes sense

Reply:
Exactly! We should still make it easy for folks to disable it and with clusterctl that means via EXP_MACHINE_POOL